### DISCUSSION TITLE
POST /auth/forgot-password 400

### DISCUSSION THREAD

**Discussion by guoyunhe** [strapi/strapi#7843] [Legacy Topics]
Hi, I added email provider and try to test reset password email:

```js
const axios = require('axios');

axios.post('http://localhost:8081/auth/forgot-password', {
  email: 'guo@adventureclub.io',
});
```

But the server says 400 Bad Request?

**Comment by derrickmehaffy** [comment]
What information does the 400 error give?

**Comment by lauriejim** [comment]
Hello! Can you please share more information?
With the current one, we can't help you.
Thank you.

**Reply by guoyunhe** [reply]
Sorry for late reply. The issue has been fixed by someone else. I don't know about the details but it seems working...

**Comment by rahsahu47** [comment]
@guoyunhe I am facing same issue. can you please provide solution?
